<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfSensePanel</class>
 <widget class="QWidget" name="SurfSensePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>523</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SurfSense</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="ExtraLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="Import">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>110</width>
          <height>48</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Import</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="ImportBtn">
           <property name="toolTip">
            <string>Import a file</string>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ProductData">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>226</width>
          <height>84</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Termék Adatok</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Matt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Fényes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_3">
           <property name="placeholderText">
            <string>Jelölje ki a koordináta rendszert...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Base">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>128</width>
          <height>42</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Bázis felvétele</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_2">
           <property name="placeholderText">
            <string>Bázis felvétele...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Measure">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>343</width>
          <height>353</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Mérés</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QPushButton" name="NewMeasureBtn">
           <property name="text">
            <string>Új mérés</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="Measurements" native="true">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Mérések:</string>
            </property>
           </widget>
           <widget class="QListView" name="MeasurementsListView">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>321</width>
              <height>131</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Sensor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>160</width>
          <height>48</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Szenzor kiválasztása</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox">
           <property name="placeholderText">
            <string>Szenzor kiválasztása...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
